### ISSUE TITLE
Administration password and username is incorrect - opencart

### ISSUE THREAD

**Issue by pulseimpulse** [bitnami/vms#711]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

dfdfdf

### bndiagnostic output

dfdfdf

### bndiagnostic was not useful. Could you please tell us why?

dfdf

### Describe your issue as much as you can

I get it , username:user and password is in the system logs, but whenever i try to log into the opencart admin page, it wont let me

**Comment by mdhont** [comment]
Please note that you need to login in the admin page, and not the customer page:

    https://IP_ADDRESS/administration

**Comment by mfatfhg** [comment]
We also downloaded the latest bitnami guacamole 1.5 some hours after it was uploaded. Its not our first appliance and we decided to replace our 1.4 bitnami guacamole with the new one.

We can access https://IP_ADDRESS or https://IP_ADDRESS:8080 and we can not login here with guacadmin/guacadmin

But we can not access https://IP_ADDRESS/administration . Its just not loading.

**Comment by pulseimpulse** [comment]
> Please note that you need to login in the admin page, and not the customer page:
> 
> ```
> https://IP_ADDRESS/administration
> ```

So what's the solution? the password and user provided in the system log is incorrect

**Comment by mfatfhg** [comment]
What i wrote two days ago reffered to bitnami guacamole v1.5.0 r01.

I mentioned that v1.5.0 r02 is available. I uploaded this OVA to our vSphere but there is no configuration wizzard during the OVA-deployment process where we can enter ssh passwords, IP-setup, etc. (which was also the case with v1.5.0 r02). So I configured the password, IP and SSH manually, but we still cant access https://IP_ADDRESS/administration and we still cant login to https://IP_ADDRESS/

Then i tried to deploy the OVA from v1.4.0 again and there is a configuration wizzard during the OVA-deploment process.
